Levels/Level.gd

```diff
@@ -49,7 +53,13 @@ func update_cover() -> void:
 			n.mark_not_visible()
 
 func spawn_enemy(spawn_pos: Vector3, enemy_type: String) -> void:
-	var enemy_instance: KinematicBody = enemy_scn.instance()
+	rng.randomize()
```

Comment:
Randomizing enemy types is not generic to all levels, since ones that are based around, eg, enemy spawn triggers will have the enemy type determined by the level designer. 
Rather, the enemy type spawned should be based on what's in the enemy_type argument.

Reply:
I've reverted the change. Sorry, I changed this as well as in testlevel just to get shotgun guys to spawn. You can best figure out how to call the proper enemy scene, maybe export loaded enemy types and be able to choose from that?

Reply:
yeah enemy spawn objects now have a string var for enemy type. probably it'd be better to change it to a dropdown menu or something
```
tool
extends SnapToTerrain
class_name EnemySpawn

export(String) var enemy_type = ""

func _ready():
	if Engine.editor_hint:
		return
	visible = false
	$EnemySpawn.use_collision = false
```